README.md

```diff
@@ -45,13 +45,18 @@ You can install CaretTab through the browser web stores
 Submit a Pull Request in GitHub if you would like to contribute. Always welcome help with translations especially :)
 
 ## Build
-Run `npm start` for a dev build. You can then run `npm run start:chrome` to start an instance of the dev build in Chrome.
+### First time installation:
+1. Make sure you have [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) installed 
+2. Run `npm install` for packages necessary to build the project
+3. Run `npm install web-ext` to be able to start a dev build instance in Chrome
```

Comment:
Should be a global install of web-ext or at least a dev dependency

Reply:
I did the same changes here as the random-theme PR, so hopefully it will make merging a little easier